signalfx-org-metrics/metrics.yaml

```diff
@@ -1,3 +1,255 @@
+
+  Dimension(s): `orgId`
+    
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansReceived
+
+sf.org.apm.numSpansDroppedThrottleByToken:
+  brief: The number of spans SignalFx receives for a specific access token
+    beyond the allowed ingest volume.
+  description: 'The number of spans SignalFx receives for a specific access token
+    beyond the allowed ingest volume. SignalFx drops spans it receives after
+    the ingestion volume limit is reached.
+
+    Dimension(s): `orgId`, `tokenId`
+    
+    Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansDroppedThrottleByToken
+
+sf.org.apm.numSpansReceivedByToken:
+  brief: The number of spans SignalFx receives for a specific access token
+    after filtering and throttling.
+  description: 'The number of spans SignalFx receives for a specific access token
+    after filtering and throttling.
+
+  Dimension(s): `orgId`, `tokenId`
+    
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansReceivedByToken
+
+sf.org.apm.grossSpanBytesReceived:
+  brief: The number of bytes SignalFx receives for a span after decompression
+    before filtering and throttling.
+  description: 'The number of bytes SignalFx receives for a span after
+    decompression before filtering and throttling.
+  
+  Dimension(s): `orgId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.grossSpanBytesReceived
+
+sf.org.apm.grossSpanBytesReceivedByToken:
+  brief: The number of bytes SignalFx receives for a specific access token
+    for a span after decompression before filtering and throttling.
+  description: 'The number of bytes SignalFx receives for a specific access token
+    for a span after decompression before filtering and throttling.
+  
+  Dimension(s): `orgId`, `tokenId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.grossSpanBytesReceivedByToken
+
+sf.org.apm.numSpanBytesReceived:
+  brief: The number of bytes SignalFx receives for a span after decompression
+    after filtering and throttling.
+  description: 'The number of bytes SignalFx receives for a span after
+    decompression after filtering and throttling.
+  
+  Dimension(s): `orgId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpanBytesReceived
+
+sf.org.apm.numSpanBytesReceivedByToken:
+ brief: The number of bytes SignalFx receives for a specific access token
+    for a span after decompression after filtering and throttling.
+  description: 'The number of bytes SignalFx receives for a specific access token
+    for a span after decompression after filtering and throttling.
+  
+  Dimension(s): `orgId`, `tokenId`
+  
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpanBytesReceivedByToken
+
+sf.org.apm.numAddSpansCallsByToken:
+  brief: The number of calls to a SignalFx ingest endpoint for a specific access token.
```

Comment:
This should be made more specific to the `/v2/trace` endpoint, or "the trace spans ingest endpoint", something like that.